icgcget/clients/icgc/icgc_client.py

```diff
@@ -22,13 +22,10 @@
 
 
 def icgc_call(object_id, token, tool_path, file_from, parallel, output, repo):
-    os.environ['ACCESSTOKEN'] = token
-    os.environ['TRANSPORT_PARALLEL'] = parallel
-    if file_from is not None:  # transport.file.from.icgc defaults to none, triggering memory mapped download
-        os.environ['TRANSPORT_FILEFROM'] = file_from
+    env = dict(os.environ, ACCESSTOKEN=token, TRANSPORT_PARALLEL=parallel, TRANSPORT_FILEFROM=file_from)
     call_args = [tool_path, '--profile', repo, 'download', '--object-id', ''.join(object_id), '--output-dir', output]
```

Comment:
It's a bit of legacy code, but `object_id` was always passed as a list to support downloading multiple files.  Icgc-storage client didn't support that, but it is passed  a list to make compatibility easier.  It has to convert to a string because subprocess arguments will only accept strings.